test/unit/custom-vocabularies-client.spec.ts

```diff
@@ -40,6 +50,30 @@ describe('custom-vocabularies-client', () => {
                 metadata
             );
 
+            expect(mockHandler.makeApiRequest).toBeCalledWith(
+                'post',
+                '',
+                { 'Content-Type': 'application/json' },
+                'json',
+                customVocabularyLegacyOptions
+            );
+            expect(mockHandler.makeApiRequest).toBeCalledTimes(1);
+            expect(customVocabularyInformation).toEqual(customVocabularyDetails);
+        });
+    });
+
+    describe('submitCustomVocabulariesNotificationConfigOnly', () => {
+        it('submit custom vocabularies with the notification config', async () => {
```

Comment:
Generally our convention for the second level of `describe` is the method name so it would still be the original method name, in `it` is where you add the different case descriptions
```suggestion
    describe('submitCustomVocabularies', () => {
        it('submit custom vocabularies with the notification config', async () => {
```